build.gradle.kts

```
@@ -96,6 +96,8 @@ dependencies {
     implementation(platform("org.openrewrite:rewrite-bom:${rewriteVersion}"))
     implementation("org.openrewrite:rewrite-java")
     implementation("org.openrewrite:rewrite-maven")
+    implementation("org.openrewrite:rewrite-github-actions")
```

```suggestion
    implementation("org.openrewrite.recipe:rewrite-github-actions:$rewriteVersion")
```

github-actions dose not have its version managed by rewrite-bom (that only covers stuff from openrewrite/rewrite) and its groupId is "org.openrewrite.recipe"

---

I was just looking at https://github.com/openrewrite/rewrite-recipe-bom/blob/main/build.gradle.kts#L36 indeed